### PR TITLE
Mira

### PR DESCRIPTION
**Pull request by SoftwareTester101**
hi girls, just updated the registration method we worked together this morning. pls have a look. thanks!

### CODE REVIEW

File: Generic/src/main/java/base/CommonAPI.java [hunk 1]
```diff
@@ -87,10 +85,10 @@
 //
 //    }
     @BeforeMethod
-
     public void setUp(){
 
-        System.setProperty("webdriver.chrome.driver", "C:\\Users\\Mira\\Drivers\\chromedriver.exe");
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\Mira\\Documents\\GitHub\\WebAutomation2019\\" +
```
Comment (anikasian) [inline Generic/src/main/java/base/CommonAPI.java]: Remind me tomorrow that we need to implement the original setUp() method.

File: Kiehls/src/main/java/com/kiehls/qa/inputdata/SearchItems.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 public class SearchItems {
 
     static ConnectToSqlDB connectToSqlDB = new ConnectToSqlDB();
+    static String filePath = "C:\\Users\\Mira\\Documents\\GitHub\\WebAutomation2019\\Kiehls\\src\\test\\resources\\secret.properties";
```
Comment (anikasian) [inline Kiehls/src/main/java/com/kiehls/qa/inputdata/SearchItems.java]: Better to it this way:
`..\\Kiehls\\src\\test\\resources\\secret.properties`

**Comment by SoftwareTester101** [comment]
Requested to merge to master by accident. Meant develop.

**Comment by SoftwareTester101** [comment]
thanks for the suggestions dear! cheers
Mira

**Pull request closed without merging**